sound/Note.java

```diff
@@ -7,66 +7,93 @@
+    public static final int HARPSICHORD =   6;
+    /**Midi-instrument Program Number for VIBES.*/
+    public static final int VIBES       =  11;
+    /**Midi-instrument Program Number for ORGAN.*/
+    public static final int ORGAN       =  19;
+    /**Midi-instrument Program Number for ACCORDIAN.*/
+    public static final int ACCORDIAN   =  23;
+    /**Midi-instrument Program Number for BANJO.*/
+    public static final int BANJO       =  25;
+    /**Midi-instrument Program Number for PIZZICATO.*/
+    public static final int PIZZICATO  =  45;
+    /**Midi-instrument Program Number for VIOLIN.*/
+    public static final int VIOLIN      =  52;
+    /**Midi-instrument Program Number for TRUMPET.*/
+    public static final int TRUMPET     =  56;
+    /**Midi-instrument Program Number for TROMBONE.*/
+    public static final int TROMBONE    =  57;
+    /**Midi-instrument Program Number for VIOLIN2.*/
+    public static final int VIOLIN_2     =  59;
+    /**Midi-instrument Program Number for VIOLIN3.*/
+    public static final int VIOLIN_3     =  65;
+    /**Midi-instrument Program Number for TENORSAX.*/
+    public static final int TENOR_SAX    =  66;
+    /**Midi-instrument Program Number for FLUTE.*/
+    public static final int FLUTE       =  73;
+    /**Midi-instrument Program Number for PANFLUTE.*/
+    public static final int PAN_FLUTE    =  75;
+    /**Midi-instrument Program Number for PIANO1.*/
+    public static final int PIANO_1      =  80;
+    /**Midi-instrument Program Number for SYNVOICE.*/
+    public static final int SYN_VOICE    =  85;
+    /**Midi-instrument Program Number for BASSDRUM.*/
+    public static final int BASS_DRUM    = 116;
+    /**Midi-instrument Program Number for MELODICTOM.*/
+    public static final int MELODIC_TOM  = 117;
+    /**Midi-instrument Program Number for SNAREDRUM.*/
+    public static final int SNARE_DRUM   = 120;
+    /** Midi Note Middle-C.*/
+    public static final int C  = 60;
+    /** Midi Minimum Note Pitch.*/
+    public static final int LOW = 25;
+    /** Midi Maximum Note Pitch.*/
+    public static final int HIGH = 120;
+    /** Midi Note Pitch Range.*/
+    public static final int RANGE = HIGH - LOW;
+    /** Whole-Note Duration (in milliseconds).*/
+    public static final int DW = 2400;
+    /** Eighth-Note Duration (in milliseconds).*/
+    public static final int DE = DW / 8;
+    /** Quarter-Note Duration.*/
+    public static final int DQ = DW / 4;
+    /** Half-Note Duration.*/
+    public static final int DH = DW / 2;
+    /** Midi Minimum Note-On Velocity - Pianissimo.*/
+    public static final int VPP = 25;
+    /** Midi Maximum Note-On Velocity - Fortissimo.*/
+    public static final int VFF = 120;
+    /** Midi Note-On Velocity Range.*/
+    public static final int VRANGE = VFF - VPP;
+    /** Special Note that represents the Null Note.*/
+    public static final Note NULL_NOTE = new Note(-1, -1, -1, -1);
 
-    // Instance Data
+    //Instance Data
```

Comment:
Given the Javadoc comments, this comment is no longer needed and should be deleted.